src/main/kotlin/service/store/StoreService.kt

```diff
@@ -26,12 +28,11 @@ class StoreService {
         val pokemonsInLocation = PokeApi.getPokemonsOfLocation(location)
         if (pokemonsInLocation.isEmpty()) return null
 
-        val price = determineBoosterpackPrice(pokemonsInLocation, location.id)
-        pokemonsInLocation.forEach { it.xp = ((it.xp * price).toInt()) }
+        val price = determineBoosterpackPrice(location.id)
 
         return Boosterpack(
             name = capitalizeLocationName(location.name),
-            price = determineBoosterpackPrice(pokemonsInLocation, location.id),
+            price = price,
```

Comment:
Uh, `price = price`... Are you sure this is correct? What does this do?

Reply:
It is a named parameter. So instead of passing everything into the constructor and relying on the correct order (which is very risky with parameters of the same type), the variable in this method just has the same name as the field in the `Boosterpack` class.

Reply:
Ah okay it looks weird, but that's just me :rofl:

Reply:
It gives safety, that's all :D